### ISSUE TITLE
no visible button in taskbar

### ISSUE THREAD

**Issue by jojopara** [MaartenBaert/ssr#601]
Hello,

As the title says it, I have no red light button in taskbar while recording.  This way I cannot be sure whether I record or not using hot keys.

I purged and re-installed.

Please help.

**Comment by jojopara** [comment]
I have just noticed that the icon is not displayed in the taskbar; only an empty space is there  functional with clicking.

**Comment by trebmuh** [comment]
Might be a duplicate of https://github.com/MaartenBaert/ssr/issues/566
What Operating System are you using?

**Comment by jojopara** [comment]
Linux MInt 18.3 Sylvia.  It showed until recently, though.

**Comment by jojopara** [comment]
I reinstalled Mint Sylvia and so far everything is all right.  It was only after reinstallation that I browsed my Mint forum posts and found the solution at https://forums.linuxmint.com/viewtopic.php?f=47&t=211693  
I hope this helps others!

**Comment by masac** [comment]
I have just noticed that the icon is not displayed in the taskbar; only an empty space is there functional with clicking. Linux Mint 18.3 Mate 64 bit, SimpleScreenRecorder 0.3.9.

**Comment by jojopara** [comment]
masac. do not blame the OS.  I am like you with Mint 18.3 and simolescreenrecorder.  I reinstalled (took me 25 min with everything) and the programme works perfectly.  We have done something that meddles with SSR, but cannot trace it.  Look at my previous post, follow the link and Cosmo's last advice!  I remember his suggestion help me.
